packages/frontend/amp/components/Expandable.tsx

```diff
@@ -0,0 +1,180 @@
+import InfoIcon from '@guardian/pasteup/icons/info.svg';
+import PlusIcon from '@guardian/pasteup/icons/plus.svg';
+
+import { body, textSans, headline } from '@guardian/pasteup/typography';
+import { palette } from '@guardian/pasteup/palette';
+
+// TODO - check if we need to use a palette colour for background (neutral 93 is
+// closest) or incorporate this one into the palette?
+const wrapper = (pillar: Pillar) => css`
+    background: #f1f1f1;
+    position: relative;
+    padding: 0 5px 6px;
+    margin: 16px 0 36px;
+
+    border-top: 13px solid ${palette.neutral[7]};
+    border-image: repeating-linear-gradient(
+            to bottom,
+            #dcdcdc,
+            #dcdcdc 1px,
+            transparent 1px,
+            transparent 4px
+        )
+        13;
+
+    a {
+        color: ${pillarPalette[pillar].dark};
+    }
+
+    p {
+        margin-bottom: 0.5rem;
+    }
+
+    b,
+    strong {
+        font-weight: bold;
+    }
+
+    ${body(2)};
+`;
+
+const buttonStyles = css`
+    height: 32px;
+    background-color: ${palette.neutral[7]};
+    border-radius: 1600px;
+    color: ${palette.neutral[100]};
+    border: none;
+    ${textSans(2)};
+    font-weight: 700;
+    padding: 0 15px 0 7px;
+
+    display: inline-flex;
+    align-items: center;
+
+    position: absolute;
+    bottom: 0;
+    transform: translate(0, 50%);
+
+    span {
+        display: inline-flex;
+        align-items: center;
+    }
+
+    svg {
+        fill: ${palette.neutral[100]};
+        width: 20px;
+        height: 20px;
+        margin-right: 10px;
+    }
+`;
+
+const headerStyle = css`
+    ${headline(3)};
+`;
+
+const creditStyle = css`
+    ${textSans(2)};
+    display: block;
+    margin: 12px 0;
+`;
+
+const pillarColour = (pillar: Pillar) => css`
+    color: ${pillarPalette[pillar].main};
+`;
+
+const headers = css`
+    margin: 0 0 16px;
+`;
+
+const innerStyle = css`
+    padding-bottom: 24px;
+`;
+
+const iconStyle = css`
+    display: inline-flex;
+    background: ${palette.neutral[60]};
+    border-radius: 100%;
+    width: 16px;
+    height: 16px;
+    align-items: center;
+    justify-content: center;
+    margin-right: 5px;
+
+    svg {
+        height: 12px;
+        fill: ${palette.neutral[100]};
+    }
+`;
+
+const imageStyle = css`
+    width: 100px;
+    height: 100px;
+    display: block;
+    float: left;
+    margin-right: 10px;
+    margin-bottom: 6px;
+
+    img {
+        object-fit: cover;
+        border-radius: 50%;
+    }
+`;
+
+export const Expandable: React.FC<{
+    id: string;
+    type: string;
+    title: string;
+    img?: string;
+    html: string;
+    credit?: string;
+    pillar: Pillar;
+}> = ({ id, type, title, img, html, credit, pillar }) => (
+    <aside className={wrapper(pillar)}>
+        <div className={headers}>
+            <span className={cx(headerStyle, pillarColour(pillar))}>
+                {type}
+            </span>
+            <h1 className={headerStyle}>{title}</h1>
```

Comment:
Pinging @SiAdcock on this one - I know this is in an aside, but I'm not keen on using `h1`s within the content in this way because there can be problems with assistive technology. *However*, my knowledge could be out of date so I'd defer to @SiAdcock on his thoughts.

Reply:
okay thanks, yeah I have no idea here.

Reply:
For the record, we've gone with h2 here, and then h3's for contained headings.